R/assertions.R

```diff
@@ -1315,22 +1317,20 @@ assert_varval_list <- function(arg, # nolint
     )]
   }
   if (length(invalids) > 0) {
-    abort(
-      paste0(
-        "The elements of the list ",
-        arg_name(substitute(arg)),
-        " must be ",
-        valid_vals,
-        ".\n",
-        paste(
-          names(invalids),
-          "=",
-          map_chr(invalids, expr_label),
-          "is of type",
-          map_chr(invalids, typeof),
-          collapse = "\n"
+    cli_abort(
+      message = message %||%
+        c(
+          paste0(
+            "The elements of the list in argument {.arg {arg_name}} must be ",
+            valid_vals, "."
+          ),
+          i = glue_collapse(
+            glue("{{.val {names(invalids)}}} = {{.code {invalids}}} is type
```

Comment:
```suggestion
            glue("{{.val {names(invalids)}}} = {{.code {invalids}}} is of type
```

Reply:
Hey @millerg23 ! I am out of office today and monday, and often won't have my computer. if you want to make small wording changes, please feel free to just push them and approve (if the wording changes satisfy all the comments that is). Thanks!